docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,346 @@ $ tctl auth rotate --type=user --grace-period=200h
+| `--google-acc`      | String containing Google service account credentials.                                                  |
+| `--google-admin`   | Email of a Google admin to impersonate.                                                                |
+| `--google-legacy`  | Flag to select groups with direct membership filtered by domain (legacy behavior). <br/>Disabled by default. [More info](https://goteleport.com/docs/enterprise/sso/google-workspace/#how-teleport-uses-google-workspace-apis) |
+| `--google-id`      | Shorthand for setting the `--id` flag to `<GOOGLE_WORKSPACE_CLIENT_ID>.apps.googleusercontent.com`     |
+
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+1. Generate OIDC auth connector configuration called `myauth`. Two mappings from OIDC claims to roles are defined:
+  - members of `admin` group will receive `access`, `editor` and `auditor` roles.
+  - members of `developer` group will receive `access` role.
+
+The values for `--secret`, `--id` and `--issuer-url` are provided by IdP.
+
+```code
+$ tctl sso configure oidc -n myauth -r groups,admin,access,editor,auditor -r group,developer,access \
+      --secret IDP_SECRET --id CLIENT_ID                                                            \
+      --issuer-url https://idp.example.com
+```
+
+2. Generate OIDC auth connector with Okta preset, enabled `groups` scope, mapping group `okta-admin` to roles `access`, `editor`, `auditor`.
+Issuer URL is set to match custom Okta domain.
+
+```code
+$ tctl sso configure oidc --preset okta --scope groups -r groups,okta-admin,access,editor,auditor \
+      --secret IDP_SECRET --id CLIENT_ID                                                          \
+      --issuer-url dev-123456.oktapreview.com
+```
+
+3. Generate OIDC auth connector with Google preset. Service account credentials are set to be loaded from /var/lib/teleport/gacc.json with --google-acc-uri.
+
+```code
+$ tctl sso configure oidc --preset google -r groups,mygroup@mydomain.example.com,access \
+      --secret SECRET --google-id GOOGLE_ID --google-acc-uri /var/lib/teleport/gacc.json \
+      --google-admin admin@mydomain.example.com
+```
+
+4. Generate the configuration and immediately test it using `tctl sso test`` command.
+
+```code
+$ tctl sso configure oidc ... | tctl sso test
+```
+
+
+### tctl sso configure saml
+
+Configure SAML auth connector, optionally using a preset.
+
+The flag `--attributes-to-roles/-r` can be provided multiple times. 
+
+To avoid errors when pasting XML to YAML file, we encourage the usage of flag `--entity-descriptor`/`-e`.
+
+```code
+$ tctl sso configure saml -e entity_desc.xml -r attr,value,role1 [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: 
```

Comment:
Since each of these flags has conditions associated with it, I think it would be cleaner to merge the mandatory flags list with the table below and mark each flag as mandatory within the table.

Reply:
Ok, I considered that way, but:
- I didn't want to make the table too wide;
- I wanted to highlight the required flags somehow.

Let me try to merge it then.

Reply:
Done. For consistency with other commands I still list the required flags before table, but details have been moved within the table itself.